ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/Accordion.js

```diff
@@ -22,6 +22,7 @@ const useStyles = makeStyles((theme) => ({
 export default function SimpleAccordion(props) {
   const classes = useStyles();
   const accordionDetailsRef = React.useRef(null);
+  const heading = props.heading.charAt(0).toUpperCase() + props.heading.slice(1);
```

Comment:
Not ideal. Too fragile. Can you defense keys by name, not index number, @harkiratsm or find a different approach?